### ISSUE TITLE
请问下该如何在src/commons/ajax.js中自定义headers到全局中？

### ISSUE THREAD

**Issue by goodliving** [zkboys/react-admin#7]
你好，我想在请求中全局添加一个token的header，但是我添加了不生效
1、在src/commons/ajax.js中添加了 sxAjax.defaults.headers.get['token'] = '1234'
2、请求
this.props.ajax.get('/api/v1/user/search', {"u": value}, {errorTip: false})
            .then(res => {
                // 如果数据为空直接返回
                if(res.data == null) {
                    return
                }
                const userList = res.data.map(user => ({
                    value: user.id,
                    label: user.display_name,
                  }));
                this.setState({userData: userList})      
            })
            .catch((err) => {
                console.log(err)
            })
            .finally(() => this.setState({fetching: false}));

3、结果没有token
GET /api/v1/user/search?u=1 HTTP/1.1
Host: localhost:4001
Connection: keep-alive
Sec-Fetch-Dest: empty
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.122 Safari/537.36
Content-Type: application/json;charset=utf-8
Accept: */*
Sec-Fetch-Site: same-origin
Sec-Fetch-Mode: cors
Referer: http://localhost:4001/task/new
Accept-Encoding: gzip, deflate, br
Accept-Language: zh-CN,zh;q=0.9

**Comment by zkboys** [comment]
sxAjax.defaults.headers['XXX-TOKEN'] = 'token-value'; 
或
sxAjax.defaults.headers.get['token'] = 'token-value';
都可以
